docs/_posts/examples/3400-01-24-drag-a-point.html

```diff
@@ -0,0 +1,166 @@
+---
+layout: example
+category: example
+title: "Create a draggable point"
+description: "Drag the point to a new location on a map and populates its coordinates in a display."
+---
+
+<style>
+.coordinates {
+    background: rgba(0,0,0,0.5);
+    color: #fff;
+    position: absolute;
+    bottom: 10px;
+    left: 10px;
+    padding:5px 10px;
+    margin: 0;
+    font-size: 11px;
+    line-height: 18px;
+    border-radius: 3px;
+    display: none;
+}
+</style>
+
+<div id='map'></div>
+<pre id='coordinates' class='coordinates'></pre>
+
+<script>
+// Holds mousedown state for events. if this
+// flag is active, we move the point on `mousemove`.
+var onDrag;
```

Comment:
The `on` prefix in javascript is usually used to indicate an event handler. Can we call this `isDragging`?